docs/framework/network-programming/tls.md

```diff
@@ -54,19 +77,64 @@ For Windows Forms and other applications, see [How to: Target a Version of the .
 
 Use the following sections to verify you're not using a specific TLS or SSL version.
 
-## If your app targets .NET Framework 4.7 or later versions
+## If you must explicitly set a Security Protocol
+
+If you must explicitly set a security protocol instead of letting .NET or the OS pick the security protocol pick these protocols:
+
+- For .NET Framework 3.5: TLS 1.2
+- For .NET Framework 4.6.2 or later: TLS 1.3
+
+If you can't find specified protocols in enum, you can add those as an extension file. See below:
+
+`SslProtocolExtensions.cs`
+
+```c#
+namespace System.Security.Authentication
+{
+    public static class SslProtocolsExtensions
+    {
+        // For .NET Framework 3.5
+        public const SslProtocols Tls12 = (SslProtocols)3072;
+        // For .NET Framework 4.6.2 and later
+        public const SslProtocols Tls13 = (SslProtocols)12288;
+    }
+}
+```
+
+`SecurityProtocolExtensions.cs`
+
+```c#
+using System.Security.Authentication;
+
+namespace System.Net
+{
+    public static class SecurityProtocolTypeExtensions
+    {
+        // For .NET Framework 3.5
+        public const SecurityProtocolType Tls12 = (SecurityProtocolType)SslProtocolsExtensions.Tls12;
+        // For .NET Framework 4.6.2 and later
+        public const SecurityProtocolType Tls13 = (SecurityProtocolType)SslProtocolsExtensions.Tls13;
+    }
+}
+```
+
+For more information, see [Support for TLS System Default Versions included in .NET Framework 3.5 on Windows 8.1 and Windows Server 2012 R2](https://support.microsoft.com/help/3154520/support-for-tls-system-default-versions-included-in-the--net-framework).
+
+## For System.Net APIs (HttpClient, SslStream)
+
+### If your app targets .NET Framework 4.7 or later versions
 
 The following sections show how to verify you're not using a specific TLS or SSL version.
 
-### For HTTP networking
+#### For HttpClient
```

Comment:
and HttpWebRequest - in .Framework the WebRequest is the main interface and HttpClient sits on top of it.